src/components/plans/details/PlanDetailsAdvancedSettingsSection.tsx

```diff
@@ -79,8 +79,7 @@ const PlanDetailsAdvancedSettingsSection = ({
                             currency: currency,
                           },
                         ),
-                        threshold.thresholdDisplayName ||
-                          `${translate('text_1724179887723pt34ivcecdt')} ${i + 1}`,
+                        threshold.thresholdDisplayName || '',
```

Comment:
I thought we should display a placeholder name here, did that changed in the meantime?

Reply:
Ye it did (cf. https://linear.app/getlago/issue/LAGO-256/%5Bqa-8%5D-remove-predefined-name-when-threshold-name-is-empty)

Reply:
ok!